src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.builds

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="Microsoft.VisualBasic.vbproj" />
+    <!-- Temporary workaround, VisualBasic projects cannot be built on non-Windows platforms. See issue #5230. -->
+    <Project Include="Microsoft.VisualBasic.vbproj" Condition="'$(OSEnvironment)' == 'Windows_NT'" />
```

Comment:
Yeah, that's why I did it. Originally I just added the OSGroup metadata, but the Ubuntu CI build (running on Windows) failed because the packaging build complained about there being missing stuff.